bootstrapvz/base/__init__.py

```diff
@@ -15,22 +15,13 @@ def validate_manifest(data, validator, error):
     from bootstrapvz.common.tools import rel_path
     validator(data, rel_path(__file__, 'manifest-schema.yml'))
 
-    from bootstrapvz.common.releases import get_release, squeeze
-    release = get_release(data['system']['release'])
-
-    if release < squeeze:
```

Comment:
Instead of removing this, I'd change it to wheezy.

Reply:
yup. you're right.